### ISSUE TITLE
Account dropdown should be keyboard navigable

### ISSUE THREAD

**Issue by brandonhaslegs** [radicle-dev/drips-app-v2#254]
The keyboard can trigger the dropdown, but can’t tab through the options.
<img width="446" alt="image" src="https://user-images.githubusercontent.com/4406983/208752478-ea611a47-0c6f-4f4f-bb8d-14a935365be3.png">

**Comment by efstajas** [comment]
Could it be a safari-only issue? I can tab through the dropdown on Chrome

**Comment by brandonhaslegs** [comment]
Yep, it’s a Safari issue
